cli/src/pcluster/config/cluster_config.py

```diff
@@ -1905,6 +1918,13 @@ def instance_type_list(self):
         """Return the list of instance types associated to the Queue."""
         return [compute_resource.instance_type for compute_resource in self.compute_resources]
 
+    def has_instance_storage(self):
```

Comment:
This duplication is actually a signal of a defect in the aforementioned class design.
By the way, this would required a refactoring that is out of scope for this PR.
So, let's proceed with the proposed implementation, but let's be sure to track the need for improvement.